lib/grpc/GrpcService.ts

```diff
@@ -119,9 +119,9 @@ class GrpcService {
    */
   public disconnect: grpc.handleUnaryCall<xudrpc.DisconnectRequest, xudrpc.DisconnectResponse> = async (call, callback) => {
     try {
-      const disconnectResponse = await this.service.disconnect(call.request.toObject());
+      await this.service.disconnect(call.request.toObject());
       const response = new xudrpc.DisconnectResponse();
-      response.setResult(disconnectResponse);
+      response.setResult('success');
```

Comment:
`Service.disconnect` to throws on failure, so there's actually no need for the response. I left it there to avoid changes in the proto file, which will be changed anyway.